unlock-app/src/__tests__/config.test.js

```diff
@@ -3,6 +3,9 @@ import {
   CANONICAL_BASE_DEV_URL,
   CANONICAL_BASE_STAGING_URL,
   CANONICAL_BASE_URL,
+  CANONICAL_STATIC_URL,
+  CANONICAL_STATIC_DEV_URL,
```

Comment:
So I am not sure this approach is right here. We should not put have different variables but a single one set based on the actual env. This would let us re-deploy with just a env var change, without a code change.

Reply:
[I know this is what happens wfor `CANONICAL_BASE_URL` but I think this is wrong too.